meta/group-info/fr.yaml

```
@@ -32,39 +29,5 @@ nat:
     structural: structure
 languages:
   partial: un peu de <language>
-  ar: Arabe
-  bg: Bulgare
-  bn: Bengali
-  ca: Catalan
-  da: Danois
-  de: Allemand
-  el: Grec
-  es: Espagnol
-  et: Estonien
-  fi: Finlandais
-  fil: Philippin
-  fr: Français
-  gl: Galician
-  hr: Croate
-  hu: Hongrois
-  id: Indonésien
-  it: Italien
-  ja: Japonais
-  ko: Coréen
-  lt: Lituanien
-  ms: Malais
-  nl: Néerlandais
-  no: Norvégien
-  pl: Polonais
-  pt: Portugais
-  ro: Roumain
-  ru: Russe
-  sr: Serbe
-  sv: Suédois
-  te: Telugu
-  th: Thaïlandais
-  tr: Turc
-  ur: Urdu
-  vi: Vietnamien
-  yue: Cantonais
-  zh: Chinois
+  partial_vowel_prefix: un peu d'<language> # This key is specific to FR
```

to match with `partial` :)

```suggestion
  partial_vowel: un peu d'<language> # This key is specific to FR
```

---

"prefix" is for "vowel prefix" cuz its when language starts with vowel